src/renderer/shared/util/sendAsset.js

```diff
@@ -16,17 +14,38 @@ export default async function sendAsset({ net, asset, amount, receiver, address,
     throw new Error(`Invalid amount: "${amount}"`);
   }
 
-  const selectedAsset = ASSETS[asset];
-  const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
-
-  const config = {
-    net,
-    address,
-    privateKey: new wallet.Account(wif).privateKey,
-    intents
-  };
-
-  const { response: { result, txid } } = await api.sendAsset(config, api.neoscan);
+  let result;
+  let txid;
+  if (keys(ASSETS).includes(asset)) {
+    const selectedAsset = ASSETS[asset];
+    const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
+
+    const config = {
+      net,
+      address,
+      privateKey: new wallet.Account(wif).privateKey,
+      intents
+    };
+
+    ({ response: { result, txid } } = await api.sendAsset(config, api.neoscan));
+  } else {
+    ({ response: { result, txid } } = await Neon.doInvoke({
```

Comment:
Can you use `api.doInvoke` here for consistency?

Reply:
Sure, will do. I just did it like in makeInvokeActions.

Reply:
Didn't realize we were using it there.  Would rather we use `api` there as well.  Thanks for pointing that out. 😄